Random-Menu.rb

```diff
@@ -0,0 +1,55 @@
+
+# a list of ten adjectives to describe food
+adj = [
+  "artisanal",
+  "soylent",
+  "creamy",
+  "fresh",
+  "spicy",
+  "tough",
+  "imaginary",
+  "raw",
+  "gluten-free",
+  "fleshy"
+]
+
+# stores 10 way to prepare food
+prep = [
+  "sauteed",
+  "deep-fried",
+  "boiled",
+  "stir-fried",
+  "baked",
+  "broiled",
+  "poached",
+  "pan-seared",
+  "steamed",
+  "mashed"
+]
+# ten foods
+
+foods = [
+  "calamari",
+  "sirloin",
+  "carrots",
+  "gouda",
+  "apple",
+  "tacos",
+  "ice cream",
+  "corndog",
+  "rice",
+  "soy nuts"
+]
+
+# instead of generating a bunch of random numbers, I am shuffling the arrays
+
+adj.shuffle!
+prep.shuffle!
+foods.shuffle!
```

Comment:
Nice use of shuffle! This is a clever way to get a random order without destroying the array as you go along.